files/tests/test_encoding.py

```
@@ -0,0 +1,91 @@
+        """Test that encoding loop doesn't run infinitely"""
+        with patch('files.tasks.FFmpegBackend') as mock_backend:
+            # Mock backend that returns unparseable output
+            mock_encoding_command = Mock()
+            mock_encoding_command.__iter__ = Mock(return_value=iter(["unparseable"] * 100))
+            mock_backend.return_value.encode.return_value = mock_encoding_command
+            
+            with patch('files.tasks.calculate_seconds', return_value=None):
+                # This test is primarily to ensure the loop exits.
+                # The actual functionality is tested by the integration test.
+                # A simple call to a mocked function within the loop will suffice.
+                with patch('files.tasks.logger.info') as mock_logger:
```

Ruff (0.14.3)

Local variable `mock_logger` is assigned to but never used

Remove assignment to unused variable `mock_logger`

(F841)